packages/ic-http-certification/src/http/http_response.rs

```diff
@@ -62,6 +62,34 @@ impl From<StatusCode> for StatusCodeWrapper {
 /// assert_eq!(response.body(), b"Hello, World!");
 /// assert_eq!(response.upgrade(), Some(false));
 /// ```
+///
+/// # Helpers
+///
+/// There are also a number of convenience methods for quickly creating an [HttpResponse] with
+/// commonly used HTTP status codes:
+///
+/// - [OK](HttpResponse::ok)
+/// - [CREATED](HttpResponse::created)
+/// - [NO_CONTENT](HttpResponse::no_content)
+/// - [MOVED_PERMANENTLY](HttpResponse::moved_permanently)
+/// - [TEMPORARY_REDIRECT](HttpResponse::temporary_redirect)
+/// - [BAD_REQUEST](HttpResponse::bad_request)
+/// - [UNAUTHORIZED](HttpResponse::unauthorized)
+/// - [FORBIDDEN](HttpResponse::forbidden)
+/// - [NOT_FOUND](HttpResponse::not_found)
+/// - [METHOD_NOT_ALLOWED](HttpResponse::method_not_allowed)
+/// - [TOO_MANY_REQUESTS](HttpResponse::too_many_requests)
+/// - [INTERNAL_SERVER_ERROR](HttpResponse::internal_server_error)
```

Comment:
Not sure whether this is not making things "too convenient", i.e. whether it is worthwhile to add so many convenience methods... 🤔  How about having at most 5 or so?  The builder-pattern is pretty easy, so it can be used directly for the less common reponse codes, while having many convenience helpers requires a substantial amount of code to maintain.

Reply:
As decided in an offline discussion we'll keep all the convenience methods for now, as apparently most of them are useful quite frequently.